src/test/java/com/uid2/admin/vertx/OperatorKeyServiceTest.java

```diff
@@ -32,18 +37,29 @@ private void checkOperatorKeyResponse(OperatorKey[] expectedOperators, Object[]
             assertEquals(expectedOperator.getProtocol(), actualOperator.getString("protocol"));
             assertEquals(expectedOperator.isDisabled(), actualOperator.getBoolean("disabled"));
             assertEquals(expectedOperator.getSiteId(), actualOperator.getInteger("site_id"));
+
+            List<Role> actualRoles = actualOperator.getJsonArray("roles").stream()
+                    .map(r -> Role.valueOf((String) r))
+                    .collect(Collectors.toList());
+            assertEquals(expectedOperator.getRoles().size(), actualRoles.size());
+            for (Role role : expectedOperator.getRoles()) {
+                assertTrue(actualRoles.contains(role));
```

Comment:
You can simplify this comparison: `assertTrue(actualRoles.containsAll(expectedOperator.getRoles()))`